mysql-deployment.yml

```diff
@@ -13,37 +13,45 @@ spec:
       nodePort: 0
   selector:
     name: mysql
-  portalIP:
-  type: ClusterIP,
+  type: ClusterIP
   sessionAffinity: None
 ---
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
+kind: DeploymentConfig
```

Comment:
The original intent of these quickstarts was that they would be Kubernetes / OpenShift agnostic. `DeploymentConfig` is an OpenShift specific construct.

I think it's ok to keep `kind: Deployment`. Also k8s does not seem to like this config so we should probably remove it:

```
  selector:
    name: mysql
```